zdm/craco/loading.py

```diff
@@ -35,14 +35,22 @@ def set_state(alpha_method=1, cosmo=Planck18):
     vparams['FRBdemo']['source_evolution'] = 0
     
     vparams['beam'] = {}
-    vparams['beam']['thresh'] = 0
-    vparams['beam']['method'] = 2
+    vparams['beam']['Bthresh'] = 0
```

Comment:
let's make sure this is updated everywhere! :)

Reply:
I wrote a script to grep for things like "'beam'" etc and it found some files under "papers" and "orig_tests". I guess you are using the "papers" section? But six further files got cleaned up and modded. I think this is everything. But if you like once this is merged back point me to your exact files and make sure